projects/fourier/lib/fourier.rb

```diff
@@ -46,9 +46,21 @@ class CLI < Thor
     desc "encrypt", "Encrypt content in the repository"
     subcommand "encrypt", Commands::Encrypt
 
-    desc "focus TARGETS", "Generate Tuist's project focusing on the target TARGET"
-    def focus(*targets)
-      Services::Focus.call(targets: targets)
+    desc "focus TARGETS", "Generate Tuist's project focusing on the targets TARGETS"
+    option(
+      :source,
+      desc: "Builds Tuist from source and uses that to focus on the targets.",
```

Comment:
I think you can simplify this, so you can use the same message everywhere
```suggestion
      desc: "Builds Tuist from sources and uses it instead of the system installed one.",
```